packages/xsnap/package.json

```diff
@@ -13,17 +13,21 @@
   },
   "scripts": {
     "repl": "node -r esm src/xsrepl.js",
-    "build": "git submodule update --init && node -r esm src/build.js",
+    "build:bundle": "rollup --config rollup.config.js",
+    "build:bin": "git submodule update --init && node -r esm src/build.js",
+    "build": "yarn build:bin && yarn build:bundle",
     "clean": "rm -rf build",
     "lint": "yarn lint:js && yarn lint:types",
-    "lint:js": "eslint 'src/**/*.js'",
+    "lint:js": "eslint 'src/**/*.js' 'lib/**/*.js'",
```

Comment:
What’s the deciding principle for what belongs in lib vs src?

Reply:
`lib` is the SES lockdown bootstrap material that we feed into rollup.

It doesn't pass `lint:types` (tons of errors from SES assert that I don't begin to understand) so I moved it somewhere that `tsc` isn't looking.

 - 1ebf699 move lockdown-shim out of src/ to avoid tsc errors